src/test/java/com/znsio/sample/e2e/screen/web/amazon/SearchResultScreenWeb.java

```diff
@@ -0,0 +1,65 @@
+package com.znsio.sample.e2e.screen.web.amazon;
+
+import com.context.TestExecutionContext;
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.entities.SAMPLE_TEST_CONTEXT;
+import com.znsio.sample.e2e.screen.amazon.ProductInformationScreen;
+import com.znsio.sample.e2e.screen.amazon.SearchResultScreen;
+import org.apache.log4j.Logger;
+import com.znsio.e2e.runner.Runner;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class SearchResultScreenWeb extends SearchResultScreen {
+
+    private static final String SCREEN_NAME = SearchResultScreenWeb.class.getSimpleName();
+    private static final Logger LOGGER = Logger.getLogger(SCREEN_NAME);
+    private final TestExecutionContext context;
+    private final Driver driver;
+    private final Visual visually;
+
+    private static final By searchResultHeader =By.xpath("//span[@data-component-type='s-result-info-bar']//descendant::span[1]");
+    private static final By productSearched =By.xpath("//span[@data-component-type='s-result-info-bar']//descendant::span[3]");
+    private static final By searchResults =By.xpath("//div[@data-component-type='s-search-result']");
+    private static final By firstProductLink =By.xpath("//div[@data-component-type='s-search-result'][1]//descendant::h2/a");
+
+    public SearchResultScreenWeb(Driver driver, Visual visually) {
+        long threadId = Thread.currentThread().getId();
+        this.driver = driver;
+        this.visually = visually;
+        this.context =Runner.getTestExecutionContext(threadId);
+    }
+
+
+    @Override
+    public boolean isUserOnSearchResultPage() {
+        LOGGER.info("On Search Results Page");
+        visually.checkWindow(SCREEN_NAME,"On Search Results Page");
+        String searchedProductCountText=driver.findElement(searchResultHeader).getText() +driver.findElement(productSearched).getText();
+        String searchedProduct=context.getTestStateAsString(SAMPLE_TEST_CONTEXT.PRODUCT_SEARCH);
+        if(searchedProductCountText.contains(searchedProduct))
+            return true;
+        return false;
```

Comment:
if(searchedProductCountText.contains(searchedProduct))
            return true;
        return false;


can be re-written as 

return searchedProductCountText.contains(searchedProduct);